langchain/agents/tools/json.ts

```diff
@@ -0,0 +1,103 @@
+import { Tool, Toolkit } from "./base";
+
+function parseInput(input: string): (string | number)[] {
```

Comment:
How about we use the JSONPointer spec, as implemented eg here https://www.npmjs.com/package/jsonpointer might even improve LLM accuracy given it's a more standard thing

Reply:
Yeah this is great. Will implement. There seems to be something similar in python too, which I can add